ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoData.java

```diff
@@ -156,12 +156,10 @@ private void computeLoopflowCnecs() {
         //TODO: when we start computing loopflows for N-1 cnecs, adapt this part of code
```

Comment:
This TODO has been done and can be removed I guess

Reply:
indeed ^^